src/kronecker.jl

```diff
@@ -0,0 +1,69 @@
+# Piracy
+const KP = Kronecker.KroneckerProduct
+_matmul!(A::KP, B::KP, C::KP) = begin
+    _matmul!(A.A, B.A, C.A)
+    _matmul!(A.B, B.B, C.B)
+    return A
+end
+_matmul!(A::KP, B::KP, C::KP, a, b) = begin
```

Comment:
This does the wrong thing! a and b should only be multiplied with one of the factors, not both! Maybe just using it on the right factors is the best here, even if that is quite some implicit behaviour?